zh/restore-data-using-tidb-lightning.md

```diff
@@ -259,28 +259,66 @@ tidb-lightning Helm chart 支持恢复本地或远程的备份数据。
         > `arn:aws:iam::123456789012:role/user` 为步骤 1 中创建的 IAM 角色。
         > ${service-account} 为 tidb-lightning 使用的 ServiceAccount，默认为 default。
 
-当 TiDB Lightning 未能成功恢复数据时，不能简单地直接重启进程，必须进行**手动干预**，否则将很容易出现错误。因此，tidb-lightning 的 `Job` 重启策略被设置为 `Never`。
+## 销毁 TiDB Lightning
 
-如果 TiDB Lightning 未能成功恢复数据，需要采用以下步骤进行手动干预：
+目前，TiDB Lightning 只能在线下恢复数据。当恢复过程结束、TiDB 集群需要向外部应用提供服务时，可以销毁 TiDB Lightning 以节省开支。
 
-1. 运行 `kubectl delete job -n ${namespace} ${release_name}-tidb-lightning`，删除 lightning `Job`。
+删除 tikv-importer 的步骤：
 
-2. 运行 `helm template pingcap/tidb-lightning --name ${release_name} --set failFast=false -f tidb-lightning-values.yaml | kubectl apply -n ${namespace} -f -`，重新创建禁用 `failFast` 的 lightning `Job`。
+* 运行 `helm delete ${release_name} --purge`。
 
-3. 当 lightning pod 重新运行时，在 lightning 容器中执行 `kubectl exec -it -n ${namespace} ${pod_name} sh` 命令。
+删除 tidb-lightning 的方法：
 
-4. 运行 `cat /proc/1/cmdline`，获得启动脚本。
+* 运行 `helm delete ${release_name} --purge`。
```

Comment:
Are the two steps the same? Why not merge them into one?

Reply:
not the same. one for tikv-importer and another for tidb-lightning.